readme.md

```diff
@@ -0,0 +1,19 @@
+[![Deploy static content to Pages](https://github.com/teehbee/social-media-client-ca/actions/workflows/pages.yml/badge.svg)](https://github.com/teehbee/social-media-client-ca/actions/workflows/pages.yml)
+
+## My submission of the course assignment for the Workflow module
+
+### Set up project
+
+<ol>
```

Comment:
HTML, in a markdown file?

Talk to GPT about how you could have avoided this :)